### ISSUE TITLE
pornhub cdn (phncdn.com)

### ISSUE THREAD

**Issue by nijaz-lab** [qsniyg/maxurl#552]
258x145: https://ci.phncdn.com/videos/202011/20/372023642/original/(m=ecuKGgaaaa)(mh=qmgGIt9NbakWB0hL)0.jpg
320x180: https://ci.phncdn.com/videos/202011/20/372023642/original/(m=eafTGgaaaa)(mh=nmla1PeM8stVe0Gg)0.jpg
not authorized: https://ci.phncdn.com/videos/202011/20/372023642/original/0.jpg
640x360: https://ci.phncdn.com/videos/202011/20/372023642/original/(m=eaAaGwObaaaa)(mh=Kas-wE--LKTsjlmh)0.jpg

https://ci.phncdn.com/videos/201805/19/166810662/original/(m=eaf8Ggaaaa)(mh=l7GhhRja7RgPmYte)1.jpg
1920x1080: https://ci.phncdn.com/videos/201805/19/166810662/original/1.jpg
zero not available: https://ci.phncdn.com/videos/201805/19/166810662/original/(m=eaf8Ggaaaa)(mh=l7GhhRja7RgPmYte)0.jpg

Looks complicated because some images say not authorized, while some work, some have zero picture, some not...

**Comment by qsniyg** [comment]
Ah thanks for that 1920x1080 URL! I thought they were all down.

It used to support this (before they added the "not authorized" for all newer URLs), but what I've done instead is make (primarily) pagelink rules (i.e. rules that require a page url, e.g. `https://imgur.com/abc` instead of `https://i.imgur.com/abc.jpg`) for pornhub, rather than simple url rules.

If I could find some way to convert the id in the url to a video id (letters and numbers), then I could create a proper url rule, which wouldn't require either a url to a pornhub video, or using the popup on the page itself. I did a bit of experimenting, but so far no luck...

**Comment by elvisef** [comment]
> Ah thanks for that 1920x1080 URL! I thought they were all down.


I don't know if it has been already fixed, but for me a link with FullHD image is showing 403:
![firefox_2021-01-14_09-16-58](https://user-images.githubusercontent.com/25976749/104562956-54308780-5649-11eb-86af-d630f257a48d.png)

Do I have to additionally authorize something here ? Maybe links are specific for some users, that do not share it for everyone.

**Comment by qsniyg** [comment]
Yeah it fails for me too, but I believe it's because the image is down ( https://ci.phncdn.com/videos/201805/19/166810662/original/(m=eaf8Ggaaaa)(mh=l7GhhRja7RgPmYte)1.jpg doesn't work for me either)
